### DISCUSSION TITLE
loss function

### DISCUSSION THREAD

**Discussion by DB1929** [amazon-science/chronos-forecasting#297] [Q&A] [answered]
Which loss function is used for pre-training Chronos-Bolt? If I want to fine-tune it, which loss function do you recommend?

**Comment by abdulfatir** [comment, marked as answer]
@DB1929 The Multi-quantile loss was used for training Chronos-Bolt. Fine-tuning is already supported in AutoGluon. Please see: https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-chronos.html#
